test/test_cuda.py

```diff
@@ -527,16 +527,6 @@ def test_cat_autogpu(self):
         z = torch.cat([x, y], 0)
         self.assertEqual(z.get_device(), x.get_device())
 
-    def test_serialization(self):
```

Comment:
Oh, so the Variable methods were duplicates too?

Yes, could you rename the test? Thank you!